library/data/component/build.gradle

```diff
@@ -0,0 +1,70 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Component Attachment API"
+	moduleName = "component"
+	id = "quilt_component"
+	description = "API allowing attachment and querying of an ECS-like interface for entities, block-entities, chunks, etc."
```

Comment:
Hmm, this description needs a bit of improvement, although I'm leaving this for my second look later